Documentation/README_TapToPlace.md

```diff
@@ -0,0 +1,95 @@
+# Tap to Place
+
+![TapToPlace](Images/Solver/TapToPlace/TapToPlaceIntroGif.gif)
+
+Tap to Place is a far interaction component that is used to place a game object on surface. This component is useful for placing objects on a spatial mesh. Tap to Place uses a combination of two clicks and head movement to place an object. A click to start the placement, head movement to change the position of the object and a click to place.
+
+## Using Tap to Place
+
+1. Set up the scene
+    - Create a new unity scene
+    - Add MRTK to the scene by navigating to the Mixed Reality Toolkit menu > Add to Scene and Configure
+    - Add a cube to the scene and change the scale to 0.2 and change the position to (0, 0, 0.7).
+1. Attach [Tap to Place](https://microsoft.github.io/MixedRealityToolkit-Unity/api/Microsoft.MixedReality.Toolkit.Utilities.Solvers.TapToPlace.html) to a game object with a collider
+
+    ![TapToPlaceInspector](Images/Solver/TapToPlace/TapToPlaceInspector2.png)
+    
+    - When the Tap to Place component is added, a Solver Handler will also be attached. Tap to Place derives from the [Solver](README_Solver.md) class which requires a Solver Handler. The position of a Tap to Place object is calculated relative to the `TrackedTargetType` within the Solver Handler. By default the Head is the `TrackedTargetType`, i.e. when the head moves, the object follows if it is selected.  The `TrackedTargetType` can also be set to Controller Ray which has the object follow the controller. The first group of properties in the Tap to Place inspector are the [Common Solver Properties](README_Solver.md#common-solver-properties).  
+    > [!NOTE]
```

Comment:
make this important. Also explain why it has to be standalone. Solvers are generally designed to be chained. But I see this one controls SolverHandler.UpdateSolvers

Reply:
added